app/(admin)/layout.tsx

```
@@ -48,168 +27,96 @@ function SidebarFallback() {
 	);
 }
 
-async function AdminLayoutContent({ children }: { children: React.ReactNode }) {
-	const session = await auth();
+async function AdminNavItems() {
+	const { hasAnyPermission, initialPermissions } =
+		await resolveAdminPermissions();
 
-	if (!session?.user?.id) {
-		notFound();
+	if (!hasAnyPermission || !initialPermissions.availableSections) {
```

Biome (2.3.13)

[error] Logical expression contains unnecessary complexity.

Safe fix: Reduce the complexity of the logical expression.


(lint/complexity/useSimplifiedLogicExpression)